bindata/network/ovn-kubernetes/common/ipsec-containerized.yaml

```diff
@@ -34,6 +34,12 @@ spec:
             - matchExpressions:
               - key: network.operator.openshift.io/dpu-host
                 operator: DoesNotExist
+{{ if .RHEL_OS_ENABLE }}
+              - key: node.openshift.io/os_id
+                operator: In
+                values:
+                - rhel
+{{ end }}
```

Comment:
also need to modify host-usr-share-rpm
on a rhel node rpm db is in `/var/lib/rpm`

and need to create the nss db directory in /var/lib/ipsec/nss 
(need a `mkdir -p` because ipsec directory is also missing)

Reply:
ok @yuvalk , now completely removing rpm check from DS manifests so that both DS can be rendered unconditionally and its pod can be scheduled accordingly based on the underlying OS.